### ISSUE TITLE
can get the repo from go

### ISSUE THREAD

**Issue by ChuckPerry** [skeema/skeema#24]
When I tried to get this repository follow the command below,It told me the wrong messages,
shell > go get github.com/skeema/skeema
/root/go/src/github.com/skeema/skeema/vendor/github.com/docker/docker/pkg/archive/archive.go:364:5: hdr.Format undefined (type *tar.Header has no field or method Format)
/root/go/src/github.com/skeema/skeema/vendor/github.com/docker/docker/pkg/archive/archive.go:364:15: undefined: tar.FormatPAX
/root/go/src/github.com/skeema/skeema/vendor/github.com/docker/docker/pkg/archive/archive.go:1166:7: hdr.Format undefined (type *tar.Header has no field or method Format)
/root/go/src/github.com/skeema/skeema/vendor/github.com/docker/docker/pkg/archive/archive.go:1166:17: undefined: tar.FormatPAX

my golang version is "go version go1.9.4 linux/amd64"
what's wrong with it?
thanks for more.

**Comment by evanelias** [comment]
Thank you for trying out Skeema and reporting this issue! It seems that one of the third-party dependencies for Skeema's test suite may now require Golang 1.10. Can you please try upgrading your version of Golang compiler and see if the error is still reported?

I believe the problem is that [github.com/docker/docker/pkg/archive](https://github.com/moby/moby/tree/master/pkg/archive) introduced a change [back in April](https://github.com/moby/moby/commit/fb170206ba12752214630b269a40ac7be6115ed4#diff-53174c8e1f0ff67b3e3cb9149146a37d) which refers to fields that were [added in Golang 1.10](https://golang.org/doc/go1.10#archive/tar).

I would like to support at least the few previous versions of Golang, but it may not be possible to unwind the dependency chain there... I'll do more digging, but in the meantime please let me know if upgrading to Golang 1.10 resolves this issue just to confirm that was the root cause. Thanks!

**Comment by evanelias** [comment]
Support for Golang 1.9 in compiling Skeema has been fixed in #27, so closing this out. Thanks again for reporting it -- I previously was not aware that dependencies had broken 1.9 support.

Moving forwards, Skeema will support the two most recent Golang versions (so currently 1.9.x and 1.10.x), which matches [Golang's own policy](https://golang.org/doc/devel/release.html#policy). Also, I've configured Travis CI to perform automated testing using 1.9.x to ensure compatibility is maintained, even though development occurs primarily using 1.10.x.
